content/blog/microcks-realm-fix-for-apicurio.md

```diff
@@ -0,0 +1,142 @@
+## Microcks Keycloak Realm fixes for APICurio Studio integration
+
+
+
+![microcks-realm-fix-for-apicurio](/images/blog/microcks-realm-fix-for-apicurio.png)
+
+Microcks is an amazing tool that helps developers mock their APIs seamlessly using OpenAPI specs. This makes it easy for distributed teams to develop complex micro-services without having to wait for full development cycles to complete, thus maximising team efficiencies.
+
+APICurio studio is another great tool to start creating your API documentations via a fully integrated OpenAPI spec editor and adds features like ability to view your documentation live as teams collaborate and edit specs on the editor in real-time.

```

Comment:
Please add a markdown link to Apicurio studio website.

Reply:
added link to APicurio Studio's website